filament/src/materials/skybox.mat

```diff
@@ -11,7 +11,8 @@ material {
         },
         {
            type : samplerCubemap,
-           name : skybox
+           name : skybox,
+           stages : [ "fragment" ]
```

Comment:
why is this change needed? Is this something we need to do more aggressively? e.g. should we tell our client to start adding this to their materials? 

Can you make a note somewhere that in theory we could inspect the vertex shader and figure this out automatically better if the stage is not specified? (maybe even file a bug).